src/server/utils/logging/config/otel.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2023 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import os from 'os';
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+// import { registerInstrumentations } from '@opentelemetry/instrumentation';
+// import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
+// import { FastifyInstrumentation } from '@opentelemetry/instrumentation-fastify';
+// import { NodeTracerProvider } from '@opentelemetry/sdk-trace-node';
```

Comment:
```suggestion
```

Reply:
@10xLaCroixDrinker can these be removed ?

Reply:
https://github.com/fastify/fastify/pull/4470 is closed (merged) and it looks like https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1679#issuecomment-1723697660 is pretty close

Reply:
I'd rather keep all of this in for now